Add functionality for extacting Lichess clock and move times

dibbles21 opened the pull request:
Hi Jason, I hope you're well!

I recently saw a [post on Reddit ](https://www.reddit.com/r/chess/comments/qjz4wq/r_package_to_parse_lichess_move_time_position/) about extracting clock times from Lichess data. I had some spare time so I have added a function to your package (`lichess_clock_move_time`) to extract columns of clock time, move time, and move number from the Lichess moves column.

The function has been tested with `testthat` and I have ran `devtools::check`, `devtools::document`, and `devtools::install`.

Any questions, advice, recommendations then please let me know.

Cheers!

Dan

JaseZiv commented:
Hi there,

Thanks for the submission!

Can I get you to update a few more files in this PR?

Can you do the following:

* Increment the version in the `DESCRIPTION` file to `1.5.0.1000`
* Add yourself as a contributor in the `DESCRIPTION` file (`Authors@R:` section)
* Update the `NEWS` doc

Let me know if you need a hand with any of it

dibbles21 commented:
Hi Jason,

Many thanks for the feedback and suggestions. I have now addressed those points in my most recent commit. Let me know if there's anything else.

dibbles21 commented:
Hi Jason. Are there any further changes you need me to make to this PR?
